bin/lms-sendinvoices.php

```diff
@@ -398,6 +400,7 @@ function localtime2($fakedate)
 		WHERE c.deleted = 0 AND d.cancelled = 0 AND d.type IN (?, ?, ?, ?)" . ($backup || $archive ? '' : " AND c.invoicenotice = 1")
             . ($archive ? " AND d.archived = 0" : '') . "
 			AND d.cdate >= $daystart AND d.cdate <= $dayend"
+            . ($ignore_send_date === "false" ? " AND d.senddate = 0" : "")
```

Comment:
Jak zrobisz to co wyżej to warunek będzie po prostu:
($ignore_send_date ? '' : " AND d.senddate = 0" )